src/scripts/load.py

```diff
@@ -0,0 +1,75 @@
+"""
+Script to be run to insert existing data from a spreadsheet to the database.
```

Comment:
I assume we're using the spreadsheet rather than the JSON files in gci-vci-aws because the spreadsheet has more info?

It's not necessarily in the scope of this script, but it would be good to make sure the JSON files in gci-vci-aws haven't diverged from the spreadsheet.

Reply:
I can add this to my "Go Live" notes when we are double checking the spreadsheet, we can cross reference the JSON files as well.